packages/devtools_app/lib/src/screens/memory/shared/heap/class_filter.dart

```diff
@@ -69,11 +69,16 @@ class ClassFilter {
   }
 
   Set<String> _filtersAsSet() {
-    Set<String> stringToSet(String? s) =>
-        s == null ? {} : s.split('\n').where((e) => e.isNotEmpty).toSet();
+    Set<String> stringToSet(String? s) => s == null
+        ? {}
+        : s
+            .split(RegExp(',|\n'))
+            .where((e) => e.isNotEmpty)
+            .map((e) => e.trim())
```

Comment:
we should `trim` before we call `where` so that we do not end up with empty strings in the set